src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -266,6 +268,14 @@ export class AppComponent implements OnInit, AfterViewChecked {
 			},
 			functionality: (batchId, resultId, index) => {
 				this.magnify(index);
+				this.dataService.sendSimpleTelemetryEvent(
+					TelemetryViewName,
+					'ResultPaneAction',
```

Comment:
why using a string here? is it throwing error if you use the enum value? if so, I think you can convert the enum definition to const enum so that it will be compiled to strings directly.

Reply:
The problem was using the enum from the telemetry file as it contained reference to an external library 'ads-extension-telemetry'. I have move the interface to its own file and fixed it for now.